src/convertToObject.js

```diff
@@ -6,7 +6,22 @@
  * @return {object}
  */
 function convertToObject(sourceString) {
-  // write your code here
+  const convertedObj = {};
+
+  sourceString.split(';').forEach((elem) => {
```

Comment:
You are splitting the string by ';' which is correct, but there can be cases where there are multiple semicolons without any styles between them or at the end of the string. This will result in empty strings in the resulting array, which can lead to unnecessary iterations. It's better to filter out these cases.